src/main/java/seedu/address/logic/commands/ChangePicCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+import javafx.stage.FileChooser;
+import seedu.address.commons.core.EventsCenter;
+import seedu.address.commons.events.ui.ProfilePictureChangedEvent;
+
+//@@author {tohcheryl}
+/**
+ * Changes the profile picture of the user
+ */
+public class ChangePicCommand extends Command {
+
+    public static final String COMMAND_WORD = "changepic";
+
+    public static final String MESSAGE_PIC_CHANGED_ACKNOWLEDGEMENT = "Profile picture has been changed!";
+
+    /**
+     * Copies profile image file set by user and saves it into images folder
+     */
+    public void copyFile(File file) throws IOException {
+        int numBytes;
+        FileInputStream fis = null;
+        FileOutputStream fos = null;
+        File outputFile = new File("src/main/resources/images/profilepic.png");
```

Comment:
As discussed in the above comment, whole `/src` is made into one JAR, so for a jar, you cant store the image inside any /src path later on.

How to solve it?

If you notice, the addressbook.xml, is stored in `data` folder, (run the jar and check how it works).
So you can try a similar way to store your profile image in `data` folder which is created in the same path as jar.